packages/braid-design-system/src/lib/components/Autosuggest/Autosuggest.tsx

```diff
@@ -768,107 +770,110 @@ export const Autosuggest = forwardRef(function <Value>(
-                                suggestion.text,
-                                value.text,
-                                suggestionHighlight,
-                              )
-                            : suggestion.highlights;
-
-                          return (
-                            <Fragment key={index + text}>
-                              {groupHeading ? (
-                                <GroupHeading>{groupHeading}</GroupHeading>
-                              ) : null}
-                              <SuggestionItem
-                                suggestion={{
-                                  ...suggestion,
-                                  highlights,
-                                }}
-                                highlighted={highlightedIndex === index}
-                                selected={value === suggestion}
-                                onClick={() => {
-                                  fireChange(suggestion);
-                                  dispatch({ type: SUGGESTION_MOUSE_CLICK });
-                                }}
-                                onHover={() => {
-                                  dispatch({
-                                    type: SUGGESTION_MOUSE_ENTER,
-                                    value: index,
-                                  });
-                                }}
-                                {...a11y.getItemProps({
-                                  index,
-                                  label: suggestion.label ?? suggestion.text,
-                                  description: suggestion.description,
-                                  groupHeading:
-                                    groupHeadingForSuggestion.get(suggestion),
-                                })}
-                              />
-                            </Fragment>
-                          );
-                        })
-                      : null}
-                  </Box>
-                </RemoveScroll>
                 {overlays}
                 {secondaryIcon}
               </Box>
             )}
           </Field>
+          <Popover
+            triggerRef={fieldRef}
+            open={isOpen}
+            width="full"
+            placement="bottom"
+            lockPlacement
+            offsetSpace="none"
+            role={false}
+          >
+            {/* MenuRef gets forwarded down to UL by RemoveScroll by `forwardProps`. */}
+            <RemoveScroll ref={menuRef} forwardProps>
+              <Box
+                textAlign="left"
+                component="ul"
+                background={
+                  !hasSuggestions && noSuggestionsMessage
+                    ? { lightMode: 'neutralSoft', darkMode: 'neutral' }
+                    : 'surface'
+                }
+                borderRadius="standard"
+                boxShadow="medium"
+                width="full"
+                marginTop="xxsmall"
```

Comment:
Maybe this should be moved to `offsetSpace` above. Have a play as i think it would be like for like